starfish/constants.py

```diff
@@ -1,29 +1,23 @@
 from enum import Enum
```

Comment:
I refactored the Enum classes so that they can be easily reused for constants in the Codebook and in IntensityTable